### PR TITLE
Tucson: Add LinearStage

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

File: apps/calsys/values-tucson-teststand.yaml [hunk 1]
```diff
@@ -3,5 +3,8 @@ namespace: calsys
 cscs:
```
Comment (mareuter) [inline apps/calsys/values-tucson-teststand.yaml:3]: Rename this directory `calsys-lab` and change any references to that within the files found in this directory. Remove the GC stuff including the `renameMap`.

**Review by mareuter — COMMENTED** [review]
Other changes:
- In `apps/cluster-config/values-tucson-teststand.yaml`, add a `calsys-lab` entry to the `namespaces` list. 
- In `apps/ospl-config/values-tucson-teststand.yaml`, add a `calsys-lab` entry to the `namespaces` list.

File: apps/calsys/values-tucson-teststand.yaml (hunk 1 above)
Comment (couger01) [reply apps/calsys/values-tucson-teststand.yaml:3]: You mean copy the folder and rename it?

**Review by mareuter — APPROVED** [review]
OK, this looks good. Let's give it a go and see what happens.

**Pull request merged**